packages/front-end/components/SyntaxHighlighting/PrismFallback.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import PrismFallback from "./PrismFallback";
+
+export default {
+  title: "Syntax Highlighting/PrismFallback",
+  component: PrismFallback,
```

Comment:
I initially wanted to do an example with the `<Code />` component but this fails to load silently, wondering if there's an exception being swallowed with the dynamic imports with the `Suspense` stuff.